TODO.md

```diff
@@ -0,0 +1,31 @@
+# TODO
```

Comment:
After failing to add client tests, I decided to make a list of things that would help improve this project, then I could select 1 or 2 for this PR. Unfortunately, because I spent so much time in dependency hell, I could only check off 1 of these.